Library/Homebrew/env_config.rb

```diff
@@ -98,15 +98,15 @@ module EnvConfig
                      "`~/.profile`, `~/.bash_profile`, or `~/.zshenv`:" \
                      '\n\n    `export HOMEBREW_CASK_OPTS="--appdir=~/Applications --fontdir=/Library/Fonts"`',
       },
+      HOMEBREW_CLEANUP_MAX_AGE_DAYS:             {
```

Comment:
A lint to ensure these are sorted some time would be nice. Either a `help wanted` issue or PR would be great